### PR TITLE
Update Port from 8302 to 8502

### PR DESCRIPTION
**Pull request by s-christoff**

### CODE REVIEW

**Comment by s-christoff** [comment]
See #5609  for more

File: website/source/docs/agent/options.html.md [hunk 1]
```diff
@@ -1724,7 +1724,7 @@ port.
 
 * DNS Interface (Default 8600). Used to resolve DNS queries. TCP and UDP.
 
-* gRPC API (Default 8302). Currently gRPC is only used to expose Envoy xDS API to Envoy proxies.
+* gRPC API (Default 8502). Currently gRPC is only used to expose Envoy xDS API to Envoy proxies.
```
Comment (pearkes) [inline website/source/docs/agent/options.html.md]: We probably want to say that it is disabled by default, but is recommended to be run on 8502, similar to what is described here:

https://www.consul.io/docs/agent/options.html#grpc_port

It's kind of a tricky port as we don't listen on it by default. Now that I'm looking at this again I wonder if we should also add `https` here as well: https://www.consul.io/docs/agent/options.html#https_port.

I think the intention of the "ports used" section is to call out network requirements specifically so I think it makes sense to cover all of the ports there (required or optional).

Comment (s-christoff) [reply website/source/docs/agent/options.html.md]: Sidecar Service Ports is odd, please let me know if you'd like me to reword it to flow more naturally.

Comment (banks) [reply website/source/docs/agent/options.html.md]: My suggestion from other PR (post mortem):

>  * gRPC API (Optional, Defaults to 8502 in `-dev` mode). Currently gRPC is only used to expose the xDS API to Envoy proxies. It is off by default but port 8502 is a convention used by various tools as the default.

Comment (banks) [reply website/source/docs/agent/options.html.md]: Maybe a similar clarification of the https port could be added but other than that I think this is fine. HTTPs is not auto-enabled in dev mode so it's not totally the same but the last sentence of the description would be appropriate there.

Comment (pearkes) [reply website/source/docs/agent/options.html.md]: I'm with @banks on the suggestions there about the off by default stuff, do you want to make that change @s-christoff?

Comment (s-christoff) [reply website/source/docs/agent/options.html.md]: I _feel_ like we are maybe being too verbose or saying default too much, this could use some minor tweaks on wording. Let me know what you guys think about the update.

Comment (banks) [reply website/source/docs/agent/options.html.md]: 👍

Comment (kaitlincarter-hc) [reply website/source/docs/agent/options.html.md]: This information was captured in https://github.com/hashicorp/consul/pull/5693.

**Comment by freddygv** [comment]
Closing, since changes were captured in #5693
